lua/frecency/db.lua

```diff
@@ -0,0 +1,186 @@
+local util = require "frecency.util"
+local const = require "frecency.const"
+local algo = require "frecency.algo"
+
+local ok, sql = pcall(require, "sql")
+if not ok then
+  error "ERROR: frecency-telescope.nvim: sql.nvim (https://github.com/tami5/sql.nvim) is not found. Please install"
+  return
+end
```

Comment:
Just saying but i think this might be obsolete soonish see https://github.com/nvim-telescope/telescope.nvim/pull/1066#discussion_r687063938